### ISSUE TITLE
new Link()->getImageLink() misses http

### ISSUE THREAD

**Issue by eschiendorfer** [thirtybees/thirtybees#1863]
I notice on my bleeding edge site: getting images by new Link(); can fail.

It returns me the url without any http:// or https://, which means that the browser adds this again leading to something like: https://www.domain.com/www.domain.com/img/bla.webp

Previously the following was working:

```
$link = new Link();
$link->getImageLink($params);
```
Since bleeding edge, I have to use:

```
$link = new Link(null, Tools::getShopProtocol());
$link->getImageLink($params);
```
or:

```
$context = Context::getContext();
$context->link->getImageLink($params);
```
Is this intended? I haven't found any issues in the core. But I guess modules could easily have the "broken" way.

**Comment by getdatakick** [comment]
There was no change. 

I've tested on 1.3.0, and the behaviour is the same.


```php
$link = new Link();
$link->getImageLink('name', 1)
```

always returned `www.domain.com/1/name.jpg`

Method `Link::getImageLink` returns 

`$this->protocol_content.Tools::getMediaServer($uriPath).$uriPath`

And `Link::$protocol_content` is only initialized in constructor using input parameter. 

This is how it was since forever. I've checked code of tb1.0.6 and it was the same.
